src/args/project-path/default.nix

```diff
@@ -7,6 +7,6 @@ if hasPrefix "/" rel
 then
   (builtins.path {
     name = if rel == "/" then "src" else builtins.baseNameOf rel;
-    path = projectSrc + rel;
+    path = ((builtins.unsafeDiscardStringContext projectSrc) + rel);
```

Comment:
On Makes with stable Nix this is:

`/tmp/tmp.JXTBpl1zAo + rel`

On Makes with unstable Nix this is:

`/nix/store/...-source + rel`

Reply:
Damn it! :smile:

Reply:
You got a more _fluid_ angle of _attack_ on the stable version of `nix` :smiley: